excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -222,7 +222,62 @@ public static String getCellValue(Workbook wb, Cell cell) {
             } else {
                 // TODO: Consider not formatting it, but simple using
                 // Double.toString(...)
-                result = _numberFormat.format(cell.getNumericCellValue());
+                result = getNumericCellValueAsStringUsingBuildinFormat(cell.getCellStyle(), cell.getNumericCellValue());
+            }
+            break;
+        case STRING:
+            result = cell.getRichStringCellValue().getString();
+            break;
+        default:
+            throw new IllegalStateException("Unknown cell type: " + cell.getCellType());
+        }
+
+        logger.debug("cell {} resolved to value: {}", cellCoordinate, result);
+
+        return result;
+    }
+
+    public static Object getCellValueAsObject(final Workbook wb, final Cell cell) {
+        if (cell == null) {
+            return null;
+        }
+
+        final String cellCoordinate = "(" + cell.getRowIndex() + "," + cell.getColumnIndex() + ")";
```

Comment:
This `String` is only used inside logging statements, for performance reasons please use format objects when logging and add the `cell.getRowIndex()` and `cell.getColumnIndex()` as arguments to the logging method.

Reply:
Actually, it's being used in so many places. I made a static method for this.